k3s_camera/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM navikey/raspbian-buster
```

Comment:
Have you tried using one of Balena's raspberry pi base images as the base instead of this? The other images in this repo use Balena base images and it would be nice if we can keep consistency. It may not be necessary to install all the additional libraries you have to install in order to get this to work, but I'm not sure, might take some poking around to see what exactly is in, e.g. the raspberrypi4-64 image.

Reply:
This works with the raspberrypi3 image, but not the `-64` image. For the 64 bit image, I get
`OSError: libmmal.so: cannot open shared object file: No such file or directory`.